mariadb-plugin/widgets/MariaDB-postaction.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
don't you forget about reference docs? (docs folder)

Reply:
yep, thx